packaging/windows/build_prerelease.cmd

```diff
@@ -15,6 +15,6 @@ popd
 
 :: Then the desktop:
 pushd  %GOPATH%\src\github.com\keybase\client\desktop
-cmd /k npm i
```

Comment:
This is weird; the script always stops here, as though some fatal exit occurs. Still working it out.